run_service.sh

```diff
@@ -999,17 +999,6 @@ export ON_CHAIN_SERVICE_ID=$service_id
 export ALL_PARTICIPANTS='["'$agent_address'"]'
 # This is the default market creator. Feel free to update with other market creators
 export OMEN_CREATORS='["0x89c5cc945dd550BcFfb72Fe42BfF002429F46Fec"]'
-export BET_AMOUNT_PER_THRESHOLD_000=0
-export BET_AMOUNT_PER_THRESHOLD_010=0
-export BET_AMOUNT_PER_THRESHOLD_020=0
-export BET_AMOUNT_PER_THRESHOLD_030=0
-export BET_AMOUNT_PER_THRESHOLD_040=0
-export BET_AMOUNT_PER_THRESHOLD_050=0
-export BET_AMOUNT_PER_THRESHOLD_060=0
-export BET_AMOUNT_PER_THRESHOLD_070=0
-export BET_AMOUNT_PER_THRESHOLD_080=30000000000000000
-export BET_AMOUNT_PER_THRESHOLD_090=80000000000000000
-export BET_AMOUNT_PER_THRESHOLD_100=100000000000000000
 export BET_THRESHOLD=5000000000000000
 export TRADING_STRATEGY=kelly_criterion
```

Comment:
Is this having any effect now after the introducion of the dynamic strategy? @Adamantios

Reply:
These are not necessary after the introduction of dynamic strategies. They are now specified in the [strategies_kwargs](https://github.com/valory-xyz/trader/blob/700af475a4538cc3d5d22caf9dec9e9d22d72af1/packages/valory/services/trader/service.yaml#L362) and can be overridden by using the `STRATEGIES_KWARGS` env var.

Reply:
I wasn't specifici enough: I meant `TRADING_STRATEGY`

Reply:
The trading strategy is ok as is. The [configuration](https://github.com/valory-xyz/trader/blob/700af475a4538cc3d5d22caf9dec9e9d22d72af1/packages/valory/services/trader/service.yaml#L82) is still the same and this will be picked from the [file_hash_to_strategies_json](https://github.com/valory-xyz/trader/blob/700af475a4538cc3d5d22caf9dec9e9d22d72af1/packages/valory/services/trader/service.yaml#L113).